Source/UI/Identity/GoldenIdentityView.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  GoldenIdentityView.swift
+//  Planetary
+//
+//  Created by Martin Dutra on 3/1/23.
+//  Copyright © 2023 Verse Communications Inc. All rights reserved.
+//
+
+import Analytics
+import CrashReporting
+import Logger
+import SwiftUI
+
+struct GoldenIdentityView: View {
+
+    var identity: Identity
+
+    var about: About?
+
+    var socialStats: SocialStats?
+
+    var hashtags: [Hashtag]?
+
+    @EnvironmentObject
+    private var botRepository: BotRepository
+
+    var body: some View {
+        VStack(alignment: .center, spacing: 0) {
+            BlobGalleryView(blobs: [Blob(identifier: about?.image?.identifier ?? .null)])
+                .allowsHitTesting(false)
+                .frame(maxWidth: .infinity, alignment: .leading)
+                .layoutPriority(1)
+            Text(about?.nameOrIdentity ?? identity)
+                .padding(EdgeInsets(top: 9, leading: 10, bottom: 5, trailing: 10))
+                .lineLimit(1)
+                .foregroundColor(.primaryTxt)
+                .font(.subheadline)
+            Text(identity.prefix(7))
+                .foregroundColor(.secondaryTxt)
+                .font(.footnote)
+                .padding(EdgeInsets(top: 0, leading: 10, bottom: 0, trailing: 10))
+            if shouldShowBio {
+                Text(bio)
+                    .frame(maxWidth: .infinity, alignment: .center)
+                    .multilineTextAlignment(.center)
+                    .allowsHitTesting(false)
+                    .padding(EdgeInsets(top: 0, leading: 10, bottom: 0, trailing: 10))
+                    .placeholder(when: about == nil) {
+                        Text(String.loremIpsum(1))
+                            .padding(EdgeInsets(top: 0, leading: 10, bottom: 0, trailing: 10))
+                            .redacted(reason: .placeholder)
+                    }
+            }
+            if let hashtags = hashtags, !hashtags.isEmpty {
+                Text(hashtags.map { $0.string }.joined(separator: " ").parseMarkdown(fontStyle: .small))
+                    .foregroundLinearGradient(.horizontalAccent)
+                    .lineLimit(1)
+                    .truncationMode(.tail)
+                    .padding(EdgeInsets(top: 0, leading: 10, bottom: 0, trailing: 10))
+            }
+            Spacer(minLength: 9)
+        }
+        .frame(maxWidth: .infinity, alignment: .leading)
+        .aspectRatio(goldenRatio, contentMode: ContentMode.fill)
+        .background(
+            LinearGradient(
+                colors: [Color.cardBgTop, Color.cardBgBottom],
+                startPoint: .top,
+                endPoint: .bottom
+            )
+        )
+        .cornerRadius(15)
+        .padding(EdgeInsets(top: 0, leading: 0, bottom: 0, trailing: 0))
+    }
+
+    private let goldenRatio: CGFloat = 0.618
```

Comment:
ah that's why it's called the GoldenIdentityView :D

Reply:
😆